templates/cube/index.html

```diff
@@ -409,22 +409,17 @@ <h3 class="p-matrix__title"><a class="p-matrix__link" href="#">Juju</a></h3>
 </section>
 
 <section class="p-strip--square-suru has-cube is-slanted--top-right">
-  <div class="row u-equal-height">
-    <div class="col-6">
-      <blockquote class="p-pull-quote">
-        <p class="p-pull-quote__quote">&nbsp;Lorem ipsum dolor sit amet, consectetur adipisicing, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam</p>
-        <cite class="p-pull-quote__citation">&mdash; A.N. Author</cite>
-      </blockquote>
-    </div>
-    <div class="col-6">
-      <blockquote class="p-pull-quote">
-        <p class="p-pull-quote__quote">&nbsp;Mauris non tempor quam, et lacinia sapien. Mauris accumsan eros eget libero posuere vulputate.</p>
-        <cite class="p-pull-quote__citation">&mdash; A.N. Author</cite>
-      </blockquote>
-    </div>
+  <div class="u-fixed-width">
+    <h2>Apply for beta access</h2>
+    <p>Canonical is seeking participants from the community for the CUBE beta test. If you would like to be considered for the beta, please enter your information in the form below. Canonical will reach out to you if you are selected.</p>
+    <button href="cube/contact-us" class="p-button--positive js-invoke-modal">Apply for access</button>
```

Comment:
I think this has to be `/cube/contact-us` or it will go to `/cube/cube/contact-us`

```suggestion
    <button href="/cube/contact-us" class="p-button--positive js-invoke-modal">Apply for access</button>
```